Makefile

```diff
@@ -416,7 +416,7 @@ build-test-system: compile-schemas update-version-file ## Build system tests doc
 
 .PHONY: package-wheel
 package-wheel: clean update-version-file ## Build python package wheel
-	python setup.py bdist_wheel
+	python -m build --wheel
```

Comment:
this requires `build` python package. add it to dev-requiremetns

Reply:
https://github.com/mlrun/mlrun/pull/4849#discussion_r1438879946